sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -34,7 +34,7 @@ static StorageTestBase()
         }
 
         public StorageTestBase(bool async, RecordedTestMode? mode = null)
-            : base(async, mode)
+            : base(async, mode, useLegacyTransport: true)
```

Comment:
Is this to stay on the old test recording framework for now?

Reply:
Yep